src/pykiso/auxiliary.py

```diff
@@ -26,9 +26,9 @@
 import time
 from typing import Any
 
-from .logging_initializer import add_logging_level
-from .test_setup.config_registry import ConfigRegistry
-from .types import MsgType
+from pykiso.logging_initializer import add_internal_log_levels
```

Comment:
Please cluster the features in the following way: one feature <-> one PR. Else, there is a miss-match between your PR title, why it exists and its content.

Reply:
That's not a feature but an extremely small refactoring at auxiliary level that just uses an API from a previous PR

I'll make a separate PR for the activate_log feature modification, but this one can definitely stay in here IMO